### PR TITLE
Minor syntax fix for const and let keywords

### PR DESCRIPTION
**Pull request by ravihara**
Corrections to the use of "const" and "let" keywords in the examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ Take a look at how this pattern may be applied in the table below.
 | `getPostData` | | `get` | `Post` | `Data` |
 | `handleClickOutside` | | `handle` | `Click` | `Outside` |
 | `shouldDisplayMessage` | `should` | `Display` | `Message`| |
+| `canAllowAnonymousUser` | `can` | `Allow` | `Anonymous`| `User` |
```
Comment (kettanaito) [inline README.md:108]: I find `can*` variables generally suitable to be substituted with the verb itself as if cast from `it`:

```diff
-canAllowAnonymoursUser
+allowsAnonymousUser
```

**Review by kettanaito — CHANGES_REQUESTED** [review]
Great fixes! Just left one comment about `canAllowAnonymousUser`.

File: README.md (hunk 1 above)
Comment (ravihara) [reply README.md:108]: Cool! Makes sense.

**Comment by kettanaito** [comment]
Hey, @ravihara. Sorry for troubling you, did you have a chance to look into the suggestions above?

**Comment by ravihara** [comment]
> Hey, @ravihara. Sorry for troubling you, did you have a chance to look into the suggestions above?

Done. I have made the necessary changes. You can try merging now. Sorry for the delay.

**Review by kettanaito — APPROVED** [review]
Thank you for this, @ravihara!

**Comment by kettanaito** [comment]
It seems this was fixed by https://github.com/kettanaito/naming-cheatsheet/pull/16.

**Pull request closed without merging**